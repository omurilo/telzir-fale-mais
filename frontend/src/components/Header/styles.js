import styled, { css } from "styled-components";

export const Header = styled.header`
  ${({ theme }) => css`
    height: 4rem;
    min-height: 4rem;
    display: flex;
    justify-content: space-between;
    align-items: center;
    padding: ${theme.spacing.horizontal};

    & > div {
      display: flex;
      align-items: center;
    }
  `}
`;
