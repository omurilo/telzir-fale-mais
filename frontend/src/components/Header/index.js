import React from "react";

import Logo from "components/Logo";

import * as Styled from "./styles";

function Header() {
  return (
    <Styled.Header>
      <Logo />
      <span>Menu?</span>
    </Styled.Header>
  );
}

export default Header;
