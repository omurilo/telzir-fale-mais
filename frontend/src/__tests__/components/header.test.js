import React from "react";
import Header from "components/Header";

import { render, screen } from "../test-utils";

describe("Header", () => {
  it("should render the Header", () => {
    render(<Header />);

    expect(screen.getByRole("banner")).toMatchSnapshot();
  });
});
