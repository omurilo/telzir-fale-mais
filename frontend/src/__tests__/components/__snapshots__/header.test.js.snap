// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the Header 1`] = `
<header
  class="styles__Header-sc-1l6xfxq-0 bLkeAo"
>
  <div>
    <span
      class="styles__LogoItem-isvygu-0 jiqBiG"
    >
      T
    </span>
    <span
      class="styles__LogoItem-isvygu-0 jiqBiG"
    >
      E
    </span>
    <span
      class="styles__LogoItem-isvygu-0 jiqBiG"
    >
      L
    </span>
    <span
      class="styles__LogoItem-isvygu-0 iUeJxp"
      color="secondary"
      rotate="25"
      scale="1.25"
    >
      Z
    </span>
    <span
      class="styles__LogoItem-isvygu-0 jiqBiG"
    >
      I
    </span>
    <span
      class="styles__LogoItem-isvygu-0 jiqBiG"
    >
      R
    </span>
  </div>
  <nav
    class="styles__Container-sc-43hkch-0 krZqfW"
  >
    <a
      class="styles__Link-sc-1hpn04p-0 styles__LinkItem-sc-43hkch-1 dJHLSH"
      href="/"
    >
      Calcula aí!
    </a>
    <a
      class="styles__Link-sc-1hpn04p-0 styles__LinkItem-sc-43hkch-1 dJHLSH"
      href="/about"
    >
      Sobre
    </a>
  </nav>
  <span
    class="styles__MenuIcon-sc-17rz6i4-1 cpjjOV null"
    role="menu"
  />
  <div
    class="styles__Container-sc-17rz6i4-0 lcWyLC"
  >
    <a
      class="styles__Link-sc-1hpn04p-0 styles__LinkItem-sc-17rz6i4-2 dJHLSH"
      href="/"
    >
      Calcula aí!
    </a>
    <a
      class="styles__Link-sc-1hpn04p-0 styles__LinkItem-sc-17rz6i4-2 dJHLSH"
      href="/about"
    >
      Sobre
    </a>
  </div>
</header>
`;
